hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/DatanodeAdminMonitorImpl.java

```diff
@@ -96,8 +97,8 @@ public class DatanodeAdminMonitorImpl implements DatanodeAdminMonitor {
   public static final class TrackedNode {
 
     private DatanodeDetails datanodeDetails;
-
     private long startTime = 0L;
+    private Map<String, List<ContainerID>> containersReplicatedOnNode = new HashMap<>();
```

Comment:
Because this can be read when it is being updated concurrently, I thiknk it needs to be a ConcurrentHashMap.